lmdeploy/vl/templates.py

```diff
@@ -250,6 +250,8 @@ class InternLMXComposer2TemplateWrapper(VLChatTemplateWrapper):
     """InternLM-XComposer2 chat template."""
 
     def append_image_token(self, prompt, num_images: int):
+        logger.warning(f'auto append {IMAGE_TOKEN} at the beginning, '
+                       'the user can manually inser the token to prompt')
```

Comment:
Typo, inser -> insert. Do we provide documents on how a user can insert the token to prompt?

Reply:
https://github.com/InternLM/lmdeploy/blob/01bb1d8132a382fdd3131cf3b643ba8f82ebc641/docs/zh_cn/multi_modal/xcomposer2d5.md#%E6%9B%B4%E5%A4%9A%E4%BD%BF%E7%94%A8%E4%BE%8B%E5%AD%90

Multi-Image